website/content/en/docs/deployment/cognito/manifest/guide.md

```diff
@@ -80,20 +80,35 @@ From this point onwards, we will be creating/updating the DNS records **only in
           export CognitoLogoutURL="https://$CognitoUserPoolDomain/logout?client_id=$CognitoAppClientId&logout_uri=$signOutURL"
           ```
 1. Substitute values for setting up Ingress.
-    1. ```bash
-        printf '
-        CognitoUserPoolArn='$CognitoUserPoolArn'
-        CognitoAppClientId='$CognitoAppClientId'
-        CognitoUserPoolDomain='$CognitoUserPoolDomain'
-        certArn='$certArn'
-        ' > awsconfigs/common/istio-ingress/overlays/cognito/params.env
-        ```
+    1. Kustomize
```

Comment:
can we use the tabpane similar to the installation command?

Reply:
Done. Let's see if it works.